code/modules/buildmode/submodes/save_area.dm

```diff
@@ -31,8 +36,13 @@
 		var/confirm = alert("Uhm, are you sure, the area is quiet large?", "Run generator", "Yes", "No")
```

Comment:
You need to check `GLOB.save_area_executing` again here in case they were sitting on the alert popup for a while and someone else started a save.

Reply:
oh I thought I moved it, must have forgot to push

Reply:
You did, I'm dumb and reviewed the comment instead of the changes.